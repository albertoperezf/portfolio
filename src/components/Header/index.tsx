import styles from "@/styles/Home.module.css";

export default function Header(): JSX.Element {
  return (
    <p>
      Alberto Perez&nbsp;
      <code className={styles.code}>Frontend Developer</code>
    </p>
  );
}
