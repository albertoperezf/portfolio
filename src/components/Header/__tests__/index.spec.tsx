import "@testing-library/jest-dom";
import { render, screen } from "@testing-library/react";
import Header from "..";

describe("Header", () => {
  it("should render the name and job title", () => {
    render(<Header />);
    expect(screen.getByText("Alberto Perez")).toBeInTheDocument();
    expect(screen.getByText("Frontend Developer")).toBeInTheDocument();
  });

  it("should match snapshot", () => {
    const { container } = render(<Header />);
    expect(container).toMatchSnapshot();
  });
});
