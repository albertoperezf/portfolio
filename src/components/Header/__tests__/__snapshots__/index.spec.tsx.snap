// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should match snapshot 1`] = `
<div>
  <p>
    Alberto PerezÂ 
    <code
      class="code"
    >
      Frontend Developer
    </code>
  </p>
</div>
`;
